zero/meta-arducam/recipes-multimedia/arducam-pivariety/arducam-pivariety_1.0.2.bb

```diff
@@ -0,0 +1,34 @@
+SUMMARY = "Arducam pivariety SDK recipe"
+SECTION = "libs"
+LICENSE = "CLOSED"
+
+SRC_URI = "\
+		file://libarducam_pivariety.so \
+		file://libarducam_pivariety.so.1.0.2 \
+		file://arducam_pivariety.hpp \
+		file://arducam_pivariety.pc \
+"
+
+INHIBIT_PACKAGE_STRIP = "1"
+INHIBIT_SYSROOT_STRIP = "1"
+INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
```

Comment:
care to share why?

Reply:
The pre-build binary is already stripped the debug information.
So if we don't set it, receive the error below.

> INHIBIT_PACKAGE_STRIP = "1"

```
ERROR: arducam-pivariety-1.0.2-r0 do_package: QA Issue: File '/usr/lib/libarducam_pivariety.so.1.0.2' from arducam-pivariety was already stripped, this will prevent future debugging! [already-stripped]
```

> INHIBIT_SYSROOT_STRIP

```
WARNING: arducam-pivariety-1.0.2-r0 do_populate_sysroot: File '/usr/lib/libarducam_pivariety.so.1.0.2' from arducam-pivariety was already stripped, this will prevent future debugging!
ERROR: arducam-pivariety-1.0.2-r0 do_package: QA Issue: File '/usr/lib/libarducam_pivariety.so.1.0.2' from arducam-pivariety was already stripped, this will prevent future debugging! [already-stripped]
```

> INHIBIT_PACKAGE_DEBUG_SPLIT

```
WARNING: arducam-pivariety-1.0.2-r0 do_populate_sysroot: File '/usr/lib/libarducam_pivariety.so.1.0.2' from arducam-pivariety was already stripped, this will prevent future debugging!
ERROR: arducam-pivariety-1.0.2-r0 do_package: QA Issue: File '/usr/lib/libarducam_pivariety.so.1.0.2' from arducam-pivariety was already stripped, this will prevent future debugging! [already-stripped]
```